plugin/main/resources/GeneratedAndroidProject/microbenchmark/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"/>
```

Comment:
In future, it would be nice to provide a way to supply a user-defined config (it may be required to setup profiling, for example).